config/software/jmxfetch.rb

```diff
@@ -1,7 +1,7 @@
 name "jmxfetch"
 
 if windows?
-  default_version "0.20.0"
+  default_version "0.20.1"
```

Comment:
I realize it was already like this, so I don't want to block the change, but why is windows hard coded an everything else from the environment?